README.md

```diff
@@ -225,6 +225,38 @@ MeasureWrite measureWrite = new MeasureWrite("sw_metric", "service_cpm_minute",
 measureBulkWriteProcessor.add(measureWrite);
 ```
 
+## Property APIs
```

Comment:
Could a tag be as a condition? The example shown here makes me feel, you are going to use one key to store all dashboards?

Reply:
If so, do these dashboards have to keep in memory, and filter based on query conditions in memory? Then in OAP cluster mode, the memory could be un-sync from time to time when users update from the web.

Reply:
<img width="78" alt="image" src="https://user-images.githubusercontent.com/5441976/163591336-c49aa4ff-f846-43d7-bf98-3b818e93c4fb.png">

These are dashboard configurations of one layer(general), and all dashboards mean all layers. You may face 35k-150k per layer, and 10+ layers right now(will add more in the future).
If the OAP Banyandb storage implementation has to load all of them to do one query, it will be an issue.

`ui-template` as property name/ID, and layer should be a tag key, we should support reading and updating one key's value per time.

Reply:
We could improve the property API in the next round. If it makes sense to us, I will fill an issue to track it.

Reply:
OK, please fix CI, and once issue filed, I am good.

Reply:
> OK, please fix CI, and once issue filed, I am good.

CI failure is caused by the Banyandb side. I am checking this issue.

Reply:
> OK, please fix CI, and once issue filed, I am good.

Everything is fine now.